src/compiler/statement/StatementedNode.ts

```diff
@@ -852,6 +853,17 @@ export function StatementedNode<T extends Constructor<StatementedNodeExtensionTy
             return this;
         }
 
+        getStructure() {
+            return callBaseGetStructure<StatementedNodeStructure>(Base.prototype, this, {
+                classes: [], // TODO: this.getClasses().map(c=>c.getStructure())
```

Comment:
This should work?

Reply:
Ah, I just realized why this is hard to do. Ok, for this just leave everything undefined. It will rely on the body text only for these kind of nodes (so source files and namespaces will only have their body text too when doing `.getStructure()`). Too difficult to handle any other way.